citadel/indico_citadel/plugin.py

```diff
@@ -51,6 +52,12 @@ class CitadelSettingsForm(IndicoForm):
                                                 'is used, the internal Indico search interface will be used. This may '
                                                 'be useful when you are still running a larger initial export and do '
                                                 'not want people to get incomplete search results during that time.'))
+    large_category_warning_threshold = IntegerField(_('Large Category Warning Threshold'),
+                                                    [NumberRange(min=0)],
+                                                    description=_('This shows a warning message to admins when '
```

Comment:
"admins" is misleading. What about something like this?

> Displays a warning to category managers when changing the ACL of big categories that would result in sending a large amount of data to the Citadel server.

FWIW this is not about incomplete search results but rather about putting high load on the infrastructure.